routes/index.js

```diff
@@ -34,6 +34,12 @@ routes.POSTlogin = function(req, res, next) {
   })(req, res, next);
 }
 
+routes.GETlogout = function(req, res){
+  console.log("logged out")
+  req.logout();
+  res.redirect("/")
```

Comment:
good point. the `res.redirect("/")` isn't actually doing anything here. We should just do `res.sendStatus(200);` to indicate a successful request.